Robot Code/src/org/usfirst/frc/team3316/robot/commands/chassis/ToggleOmni.java

```diff
@@ -0,0 +1,47 @@
+package org.usfirst.frc.team3316.robot.commands.chassis;
+
+import org.usfirst.frc.team3316.robot.commands.DBugCommand;
+
+public class ToggleOmni extends DBugCommand
+{
+	private static boolean set = true;
+	
+	private DBugCommand extend, retract;
+	
+	public ToggleOmni ()
+	{
+		extend = new ExtendOmni();
+		retract = new RetractOmni();
+	}
+	
+	protected void init()
+	{
+		if (set)
+		{
+			extend.start();
+		}
+		else
+		{
+			retract.start();
+		}
+		
+		set = !set;
```

Comment:
Note what I wrote in `ExtendOmni:init`:
Maybe we can get this state from a sensor and the piston states instead of keeping it? would it be better? (I'm not sure).